pkgs/applications/version-management/git-and-tools/git-machete/default.nix

```diff
@@ -21,7 +22,7 @@ buildPythonApplication rec {
 
   nativeBuildInputs = [ installShellFiles ];
 
-  checkInputs = [ git ];
+  checkInputs = [ git pytest ];
```

Comment:
Will be needed soon to release another version (3.9.1 or 3.10.0, whatever it's gonna be)

Reply:
in nixpkgs we usually use `pytestCheckHook` so that the tests are run automatically without having to manually do them in checkPhase

Reply:
Done, reworked &mdash; pls check now